apiextensions/storageversion/migrator.go

```diff
@@ -55,11 +56,13 @@ func NewMigrator(d dynamic.Interface, a apixclient.Interface) *Migrator {
 // Finally the migrator will update the CRD's status and drop older storage
 // versions
 func (m *Migrator) Migrate(ctx context.Context, gr schema.GroupResource) error {
+	logger := logging.FromContext(ctx)
 	crdClient := m.apixClient.ApiextensionsV1().CustomResourceDefinitions()
 
 	crd, err := crdClient.Get(ctx, gr.String(), metav1.GetOptions{})
 	if err != nil {
-		return fmt.Errorf("unable to fetch crd %s - %w", gr, err)
+		logger.Infof("Ignoring resource migration - unable to fetch crd %s - %w", gr, err)
+		return nil
```

Comment:
The ignore should only happen when `apierrors.IsNotFound(err)` otherwise migration errors aren't visibles.

Reply:
Is it possible to add a test?

Reply:
Yes. Definitely. Thank you!